packages/ui-extensions/src/surfaces/point-of-sale/api/print-api/print-api.ts

```diff
@@ -3,9 +3,9 @@
  */
 export interface PrintApiContent {
   /** Trigger a print dialog
-   * @param src the source to print. Omit to use the current PrintPreview contents.
+   * @param src the source URL of the content to print. A src is required regardless if you are using the PrintPreview component.
```

Comment:
I'd suggest adding the requirements here and in the PrintPreview component, just so it's written down somewhere. This might be an example:

```suggestion
   * @param src the source URL of the content to print. This URL must be a path of the app backend. Valid document types are text, HTML, image, and PDF.
```